docs/pages/machine-id/workload-identity.mdx

```diff
@@ -0,0 +1,137 @@
+string that identifies your workload within the trust domain. What you include
+within this path is up to you and your application's requirements. Typically,
+the hierarchical nature of the path is leveraged. For example, you had the
+service `foo` operating in the `europe` region, you may wish to represent this
+as: `/region/europe/svc/foo`.
+
+Together, this produces a SPIFFE ID that looks like:
+
+```
+spiffe://example.teleport.sh/region/europe/svc/foo
+```
+
+### Secure Verifiable Identity Documents (SVIDs)
+
+The SPIFFE ID may be a unique identifier for a workload, but provides no way
+for a workload to verifiably prove its identity. This is where the Secure
+Verifiable Identity Documents (SVIDs) come in.
+
+The SVID is a document that encodes the SPIFFE ID and a cryptographic proof
+which allows the SVID to be verified as issued by a trusted authority.
+
+SPIFFE sets out two formats for SVIDs:
+
+- X.509-SVID: These are X.509 certificates that include the SPIFFE ID encoded in
+  the URI SAN field. This certificate is then signed by a trusted authority
+  within the trust domain.
+- JWT-SVID: These are JWT tokens that include the SPIFFE ID as the `sub` claim.
+  These are signed by a trusted authority within the trust domain.
+
+The data needed by a workload to verify a SVID is known as the trust bundle.
+This is a set of certificates belonging to the trusted authorities within the
+trust domain.
+
+### Workload API
+
+### Teleport's Workload Identity
+
+Teleport's Workload Identity is an implementation of SPIFFE. Each Teleport
+cluster acts as a SPIFFE trust domain, with the Auth Service as a certificate
+authority for issuing SVIDs.
+
+Teleport's RBAC system is used to control which Bots and Users are able to
+request a SVID for a given SPIFFE ID. Roles can specify which SPIFFE IDs can
+be issued and this role is then granted to the Bot or User. For example:
+
+```yaml
+kind: role
+version: v6
+metadata:
+  name: europe-foo-svid-issuer
+spec:
+  allow:
+    spiffe:
+    - path: "/region/europe/svc/foo"
+```
+
+The SPIFFE Workload API is implemented as a configurable service within the
+`tbot` agent. The `tbot` agent should be installed close to the workloads that
+need to request SVIDs, and they can then use the Workload API exposed by `tbot`
+to fetch SVIDs and Trust Bundles.
+
+Teleport's Workload Identity currently only supports issuing X.509-SVIDs.
+
+## Next Steps
+
+- [Getting Started](./workload-identity/getting-started.mdx): How to configure Teleport for Workload Identity.
+- [Best Practices](./workload-identity/best-practices.mdx): Best practices for using Workload Identity in Production.
+- [TSH Support](./workload-identity/tsh.mdx): How to use `tsh` with Workload Identity to issue SVIDs to users.
+
+### Other Resources
+
+- [SPIFFE Specification](https://github.com/spiffe/spiffe/blob/main/standards/SPIFFE.md): The official SPIFFE specification. Useful for understanding the SPIFFE ID and SVID formats.
```

Comment:
I would move this link to where we first introduce SPIFFE. In a conceptual guide like this, I would expect to find a link to the concept when first reading about the concept.

Reply:
ae32679d56